draft-ietf-quic-http.md

```diff
@@ -472,87 +475,86 @@ or complete response but does not abort reading, clients SHOULD continue sending
+lowercase prior to their encoding.  A request or response containing uppercase
+characters in field names MUST be treated as malformed ({{malformed}}).
 
 Like HTTP/2, HTTP/3 does not use the Connection header field to indicate
-connection-specific header fields; in this protocol, connection-specific
-metadata is conveyed by other means.  An endpoint MUST NOT generate an HTTP/3
-message containing connection-specific header fields; any message containing
-connection-specific header fields MUST be treated as malformed ({{malformed}}).
+connection-specific fields; in this protocol, connection-specific metadata is
+conveyed by other means.  An endpoint MUST NOT generate an HTTP/3 field section
+containing connection-specific fields; any message containing
+connection-specific fields MUST be treated as malformed ({{malformed}}).
 
 The only exception to this is the TE header field, which MAY be present in an
-HTTP/3 request; when it is, it MUST NOT contain any value other than "trailers".
+HTTP/3 request header; when it is, it MUST NOT contain any value other than
+"trailers".
 
 This means that an intermediary transforming an HTTP/1.x message to HTTP/3 will
-need to remove any header fields nominated by the Connection header field, along
-with the Connection header field itself.  Such intermediaries SHOULD also remove
-other connection-specific header fields, such as Keep-Alive, Proxy-Connection,
+need to remove any fields nominated by the Connection field, along with the
+Connection field itself.  Such intermediaries SHOULD also remove other
+connection-specific fields, such as Keep-Alive, Proxy-Connection,
 Transfer-Encoding, and Upgrade, even if they are not nominated by the Connection
-header field.
+field.
 
 #### Pseudo-Header Fields
 
-As in HTTP/2, HTTP/3 uses special pseudo-header fields beginning with the ':'
-character (ASCII 0x3a) to convey the target URI, the method of the request, and
-the status code for the response.
+As in HTTP/2, HTTP/3 uses a series of pseudo-header field lines beginning with
+the ':' character (ASCII 0x3a) to convey the target URI, the method of the
+request, and the status code for the response.
 
-Pseudo-header fields are not HTTP header fields.  Endpoints MUST NOT generate
+Pseudo-header fields are not HTTP fields.  Endpoints MUST NOT generate
 pseudo-header fields other than those defined in this document, except as
 negotiated via an extension; see {{extensions}}.
 
-Pseudo-header fields are only valid in the context in which they are defined.
-Pseudo-header fields defined for requests MUST NOT appear in responses;
-pseudo-header fields defined for responses MUST NOT appear in requests.
-Pseudo-header fields MUST NOT appear in trailers.  Endpoints MUST treat a
-request or response that contains undefined or invalid pseudo-header fields as
-malformed ({{malformed}}).
+Pseudo-fields are only valid in the context in which they are defined.
```

Comment:
That's what it was in an earlier version of this PR; the pushback was that these only occur in the header, not in trailers, so it's more precise to say header fields than simply fields.  I agree that "pseudo-header field" makes it sound like there's a region called the "pseudo-header," and there kind of is since these have to be first, but we don't call it that.